update kafka docs and defaults to v2.0.0 (#1221)
grafana/metrictank: docs/kafka.md

# Kafka version

> We recommend v2.0.0 or higher, as that is what we use.

Reviewer: ```suggestion
Kafka v2.0.0 or newer is recommended.
```
